### PR TITLE
Updated Lightbend company name

### PR DESCRIPTION
**Pull request by Silv3rcircl3**

### CODE REVIEW

File: src/benchmark/PersistenceBenchmark/Messages.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright file="Messages.cs" company="Akka.NET Project">
-//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.Lightbend.com>
```
Comment (sean-gilliam) [inline src/benchmark/PersistenceBenchmark/Messages.cs]: Not that it makes a to much of a difference, but shouldn't that be http://www.lightbend.com

**Comment by Silv3rcircl3** [comment]
@sean-gilliam done

**Comment by Danthar** [comment]
@Silv3rcircl3 Can you rebase this pr.

**Comment by Silv3rcircl3** [comment]
@Danthar dibe

**Pull request merged**